src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -88,6 +88,9 @@ public AnalyzeCommandLineOption(Options options)
                 TargetMapFile = options.TargetMap;
                 BreakingChangeSuppressions = options.SuppressBreakingChange;
 
+                //Set OverwriteOutputFile to true if the output file name is explicitly specified 
+                OverwriteOutputFile = string.IsNullOrWhiteSpace(options.Out) ? false : true;
```

Comment:
I've replaced the code fragment with what you suggested. If there are invalid characters, `Path.GetFullPath(string)` will fail with the error message `Illegal characters in path` . It will also fail for an input like `c:\temp:\myfile` , which checking for invalid characters won't catch. Currently for failure of `Path.GetFullPath()` the error message displayed is `There was an unexpected error.  Please try again or contact NETAPIPort@Microsoft.com for help.`,  so I put it under a try/catch and display the error message instead.